client/src/store/modules/eventTypes/mutations.js

```diff
@@ -0,0 +1,17 @@
+import * as mutations from './types/mutations';
+
+export default {
+    [mutations.SET_EVENT_TYPES]: (state, eventTypes) => {
+        state.eventTypes = eventTypes;
+    },
+    [mutations.DISABLE_EVENT_TYPE_BY_ID]: (state, data) => {
+        state.eventTypes.find(eventType => {
+            return eventType.id === data.id;
+        }).disabled = data.disabled;
```

Comment:
I didn't notice this, find returns either found item or `undefined`. So if there is not eventType with data.id, the uncaught exception will be fired